packages/engine/src/utils/engineEnvBuilder.ts

```
@@ -758,18 +764,15 @@ function configureIdentity(coreConfig: IEngineConfig, envVars: IEngineEnvironmen
 			type: IdentityConnectorType.EntityStorage
 		});
 	} else if (envVars.identityConnector === IdentityConnectorType.Iota) {
-		coreConfig.types.identityConnector.push({
-			type: IdentityConnectorType.Iota,
-			options: {
-				config: {
-					clientOptions: {
-						url: envVars.iotaNodeEndpoint ?? ""
-					},
-					network: envVars.iotaNetwork ?? "",
-					coinType: Coerce.number(envVars.iotaCoinType)
+		const iotaConfig = getIotaConfig(coreConfig);
+		if (iotaConfig) {
```

This `if` should not be here, we need the configuration to always be added, otherwise the Guards within the constructors will never be triggered and it will just fail silently.